ports/psoc6/main.c

```diff
@@ -67,6 +88,7 @@ int main(int argc, char **argv) {
 
     #endif /* #if defined (CY_DEVICE_SECURE) */
 
+
     /* Initialize the device and board peripherals */
     result = cybsp_init();
```

Comment:
Similar to https://github.com/jaenrig-ifx/micropython/pull/17/files#r1049659473 .